src/main/scala/housekeeper/AWS.scala

```diff
@@ -0,0 +1,18 @@
+package housekeeper
+
+import com.amazonaws.auth.{AWSCredentialsProviderChain, EC2ContainerCredentialsProviderWrapper}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.regions.Regions.EU_WEST_1
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
+import com.amazonaws.services.sns.AmazonSNSAsyncClient
+
+object AWS {
+  val credentials = new AWSCredentialsProviderChain(
+    new EC2ContainerCredentialsProviderWrapper,
```

Comment:
This should have been `EnvironmentVariableCredentialsProvider`, not `EC2ContainerCredentialsProviderWrapper` (see eg https://stackoverflow.com/a/32410565/438886 or https://github.com/guardian/mobile-n10n/pull/156 - I got it wrong, back in October 2019) - which meant the lambda couldn't load the AWS credentials it needed to delete entries from the [`ophan-alerts`](https://eu-west-1.console.aws.amazon.com/dynamodb/home?region=eu-west-1#tables:selected=ophan-alerts;tab=items) DynamoDB table, or post to the [SNS topic](https://eu-west-1.console.aws.amazon.com/sns/v3/home?region=eu-west-1#/topic/arn:aws:sns:eu-west-1:021353022223:Ophan-Housekeeper-PermanentEmailBounceTopic-13TE98BK31YQJ)... :crying_cat_face: 

Perhaps surprisingly, the [Ophan Housekeeper lambda](https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions/Ophan-Housekeeper-Lambda-KBZN4AWWA0W7) _only_ needs those AWS credentials when it's dealing with a permanently bouncing email - so there was no obvious problem until a permanent bounce occurred, with James Murray's Ophan Alert emails bouncing, starting at [13:26 on February 12th 2020](https://logs.gutools.co.uk/s/ophan/app/kibana#/doc/581f7b00-c4df-11e9-bc08-9d1af4d9c1d5/logstash-ophan-2020.02.12?id=ZjiUOXABTFIJeJSxqfmH&_g=()).

At that point these errors [started occurring](https://logs.gutools.co.uk/s/ophan/app/kibana#/doc/581f7b00-c4df-11e9-bc08-9d1af4d9c1d5/logstash-ophan-2020.02.12?id=ciyUOXABBviLcZqRrCAI&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2020-02-12T13:26:31.042Z',to:'2020-02-12T13:26:40.791Z'))):

```
Unable to load AWS credentials from any provider in the chain: [com.amazonaws.auth.EC2ContainerCredentialsProviderWrapper@1e730495: Unable to load credentials from service endpoint, com.amazonaws.auth.profile.ProfileCredentialsProvider@7d3a22a9: profile file cannot be null]: com.amazonaws.SdkClientException
com.amazonaws.SdkClientException: Unable to load AWS credentials from any provider in the chain: [com.amazonaws.auth.EC2ContainerCredentialsProviderWrapper@1e730495: Unable to load credentials from service endpoint, com.amazonaws.auth.profile.ProfileCredentialsProvider@7d3a22a9: profile file cannot be null]
	at com.amazonaws.auth.AWSCredentialsProviderChain.getCredentials(AWSCredentialsProviderChain.java:136)
...
	at com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient.doInvoke(AmazonDynamoDBClient.java:4805)
	at com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient.invoke(AmazonDynamoDBClient.java:4772)
	at com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient.executeQuery(AmazonDynamoDBClient.java:2641)
	at com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient.query(AmazonDynamoDBClient.java:2607)
	at org.scanamo.ops.ScanamoSyncInterpreter.apply(ScanamoSyncInterpreter.scala:35)
```

As the lambda was broken, it wasn't able to decommission James Murray's Ophan Alerts - so we are now under review:

https://groups.google.com/a/guardian.co.uk/d/msg/aws-ophan/F880jhVL2yY/Wz_iVkkZAQAJ